src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -724,6 +739,115 @@ public PrefixEnumerable FindKeysWithPrefix(string prefix)
             return new PrefixEnumerable(this, prefix);
         }
 
+        [DebuggerDisplay("SubKey={SubKey}, Key={Key}, State={ValidationState}")]
```

Comment:
Will be confusing in the debugger unless you use `Visible` e.g. `{Visible ? ValidationState : "hidden"}`